spec/deps/git.go

```diff
@@ -167,6 +166,7 @@ func match(p string, exp string) (gs *Git, version string) {
 		gs.Host = matches["host"]
 		gs.User = matches["user"]
 		gs.Repo = matches["repo"]
+		gs.Scheme = matches["scheme"]
```

Comment:
This is overridden anyways .. for SSH with `GitSchemeSSH` and for HTTPS with `GitSchemeHTTPS`.

I'd say just remove the whole scheme thing from the regex and handle it in the switch above

Reply:
:+1: Done deal.